libs/backend/src/ballot_package/ballot_package_io.test.ts

```diff
@@ -25,6 +25,7 @@ test('readBallotPackageFromUsb can read a ballot package from usb', async () =>
     user: fakeElectionManagerUser({
       electionHash: electionDefinition.electionHash,
     }),
+    sessionExpiresAt: new Date().getTime() + 60 * 1000,
```

Comment:
This is of course fine as is, but thought I'd mention that test-utils provides a `fakeSessionExpiresAt()` helper